p2p/net/mock/mock_net.go

```diff
@@ -108,14 +109,16 @@ func (mn *mocknet) AddPeer(k ic.PrivKey, a ma.Multiaddr) (host.Host, error) {
 }
 
 func (mn *mocknet) AddPeerWithPeerstore(p peer.ID, ps peerstore.Peerstore) (host.Host, error) {
-	n, err := newPeernet(mn, p, ps)
+	bus := eventbus.NewBus()
+	n, err := newPeernet(mn, p, ps, bus)
 	if err != nil {
 		return nil, err
 	}
 
 	opts := &bhost.HostOpts{
 		NegotiationTimeout:      -1,
 		DisableSignedPeerRecord: true,
+		EventBus: bus,
 	}
 
 	h, err := bhost.NewHost(n, opts)
```

Comment:
Any reason _not_ to use lightweight blankhost here isntead?
Doesn't seem like basichost gives anything here

Reply:
Generally I'd prefer this to behave as similar to a normal node as possible. So I'd lean towards keeping this as the basic host. Is there something in basichost that's too heavy for you? I wonder if it would make more sense to make that lighter instead.

Reply:
Nothing specific, just wonderting. BasicHost also works